.travis.yml

```diff
@@ -6,3 +6,8 @@ jdk:
 cache:
   directories:
     - $HOME/.m2/repository
+install: true
+script:
+  - mvn clean verify
```

Comment:
Travis has two steps for any build.
An install step to download all the dependencies and a script step.
https://docs.travis-ci.com/user/job-lifecycle/

For java maven builds the install step is not needed since we are not installing any rpms/npm packages/pip packages.
script step should cover the `prepare-package` step since that is where we are hooking jacoco 
See https://github.com/prestodb/airbase/blob/master/airbase/pom.xml#L795-L799
So I have decided to go with mvn clean verify which will cover that step as well as running any integration tests and stops just short of installing the built modules. `mvn clean install` seemed like an overkill since we are not too concerned about installing the built modules to travis's local repo.